### PR TITLE
[SWATANG-10] - implement add skill profile

### PR DESCRIPTION
**Pull request by thienkieu**
### Description
- Implement new feature to add skill profile 
- Implement unit test for feature component

### ☑️ Definition of Done Checklist
- [x] Keep the application can be launched successfully
- [x] Keep the source code is built successfully
- [x] Keep the unit test can run successfully

### Check the box when resolved these items and provide more information

- [ ] Documentation
- [ ] Localization
- [x] Coding
- [x] Unit Tests
- [ ] Integration Tests
- [ ] System Tests

### 📷 Screenshot / GIF

UI

test1

### CODE REVIEW

File: src/polyfills.ts [hunk 1]
```diff
@@ -77,7 +77,7 @@
 /***************************************************************************************************
  * Zone JS is required by default for Angular itself.
  */
-import 'zone.js/dist/zone' // Included with Angular CLI.
+import 'zone.js/dist/zone.js' // Included with Angular CLI.
```
Comment (thienkieu) [inline src/polyfills.ts:80]: fllowing the suggestion at https://ckeditor.com/docs/ckeditor5/latest/builds/guides/integration/frameworks/angular.html#zonejs to fix unitest

File: src/app/shared/validation/validation.service.ts [hunk 2]
```diff
@@ -6,9 +6,15 @@ export class ValidationService {
   private fieldValidationMessages = {
     usernameMinLength: 'Minimum length of username must be 6 characters.',
     usernameRequired: 'Username is required.',
+    skillNameRequired: 'Skill name is required.',
+    skillSchemaNameRequired: 'Skill schema name is required.',
     usernamePattern: 'Username must contain only alphabetic characters.',
     nameRequired: 'Name is required.',
     nameMaxLength: 'The maximum length of name is 50 characters.',
+    skillNameMaxLength: 'The maximum length of name is 100 characters.',
```
Comment (eisen-develops) [inline src/app/shared/validation/validation.service.ts]: a skill name

File: src/app/shared/validation/validation.service.ts [hunk 3]
```diff
@@ -6,9 +6,15 @@ export class ValidationService {
...
+    skillLevelDescription: 'The maximum length of description is 2000 characters.',
```
Comment (eisen-develops) [inline src/app/shared/validation/validation.service.ts]: a skill level description

File: src/app/shared/validation/validation.service.ts [hunk 4]
```diff
@@ -6,9 +6,15 @@ export class ValidationService {
...
+    skillSchemaNameLength: 'The maximum length of skill schema name is 100 characters.',
+    skillLevelName: 'The maximum length of name is 50 characters.',
```
Comment (eisen-develops) [inline src/app/shared/validation/validation.service.ts]: a skill level name

File: src/app/shared/validation/validation.service.ts (hunk 2 above)
Comment (thienkieu) [reply src/app/shared/validation/validation.service.ts]: Hi @1ceb3rg fixed, Please check it again

File: src/app/shared/validation/validation.service.ts (hunk 3 above)
Comment (thienkieu) [reply src/app/shared/validation/validation.service.ts]: Hi @1ceb3rg fixed, Please check it again

File: src/app/shared/validation/validation.service.ts (hunk 4 above)
Comment (thienkieu) [reply src/app/shared/validation/validation.service.ts]: Hi @1ceb3rg fixed, Please check it again

**Review by mwd-luongmai — APPROVED** [review]
LGTM

**Review by quangloc1993 — APPROVED** [review]
LGTM

**Pull request merged**